keanu-project/build.gradle

```diff
@@ -149,8 +149,12 @@ task sourcesJar(type: Jar) {
     from sourceSets.main.allSource
 }
 
+task deletePythonClasspath(type: Delete) {
+    delete fileTree("../keanu-python/keanu/classpath/")
+}
```

Comment:
Nitpick: can we put a newline after the task?

Reply:
done